web/css/ToDoListStyleSheet.css

```diff
@@ -108,4 +115,9 @@ body{
     overflow: auto;
     padding: 5px;
     margin-left: 40%;
```

Comment:
margin-left: 40% is evil. it doesnt work on all screen resolutions. please come up with a different and elegant solution.